src/thor/triplegbuilder.cc

```diff
@@ -191,35 +201,45 @@ void SetShapeAttributes(const AttributesController& controller,
     cuts.emplace_back(cut_t{tgt_pct, speed, UNKNOWN_CONGESTION_VAL, {}});
   }
 
-  // sort the start and ends of the incidents along this edge
-  for (const auto& incident : incidents) {
-    // if the incident is actually on the part of the edge we are using
-    if (incident.start_offset > tgt_pct || incident.end_offset < src_pct)
-      continue;
-    // insert the start point and end points
-    for (auto offset : {
-             std::max(incident.start_offset, src_pct),
-             std::min(incident.end_offset, tgt_pct),
-         }) {
-      // if this is clipped at the beginning of the edge then its not a new cut but we still need to
-      // attach the incidents information to the leg
-      if (offset == src_pct) {
-        UpdateIncident(leg, &incident, shape_begin);
-        continue;
+  if (incidents.tile) {
```

Comment:
not needed

Reply:
if you really want it though you could put it in the while part of the for loop below